.github/workflows/main.yaml

```diff
@@ -349,3 +351,57 @@ jobs:
         run: |
           export DOCKER_BUILD_AND_SYSTEM_PRUNE="true"
           make image 
+  container_runtime_test:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        container-runtime: [ 'cri-o', 'docker' ]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install and configure container runtime
```

Comment:
Is there any method to write the runtime installation steps into the script file of hack directory instead of in the workflow?

Reply:
This can be written in the script under hack, and I will place the installation of CRI in install.sh.